src/core/callouts/qgscallout.h

```diff
@@ -81,6 +82,15 @@ class CORE_EXPORT QgsCallout
       OrderBelowIndividualLabels, //!< Render callouts below their individual associated labels, some callouts may be drawn over other labels
     };
 
+    //! Feature's anchor point position
+    enum AnchorPoint
+    {
+      PoleOfInaccessibility = 0, //!< The surface's pole of inaccessibility used as anchor
```

Comment:
Might be worth marking these as polygon only options in the dox (planning ahead for future line anchor options)